src/main/java/org/jenkinsci/test/acceptance/po/Trigger.java

```diff
@@ -2,8 +2,7 @@
 
 /**
  * Base type for {@link PageAreaImpl} for trigger.
- *
- * @see {@link Job#addTrigger(Class)}.
+ * See {@link Job#addTrigger(Class)}.
```

Comment:
Why is this not using @see as that is the purpose of it?

```suggestion
 * @see Job#addTrigger(Class)
```